tests/_passport/test_passportelementerrorfiles.py

```diff
@@ -58,12 +59,17 @@ def test_to_dict(self, passport_element_error_files):
         assert isinstance(passport_element_error_files_dict, dict)
         assert passport_element_error_files_dict["source"] == passport_element_error_files.source
         assert passport_element_error_files_dict["type"] == passport_element_error_files.type
-        assert (
-            passport_element_error_files_dict["file_hashes"]
-            == passport_element_error_files.file_hashes
-        )
```

Comment:
making it this a property means making `file_hashes` private and thus it is removed from `to_dict`. Should I override to_dict to correct this then?